File: test/fixtures/php-guzzle/post_json.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->post('http://localhost:28139', [
    'headers' => [
        'Content-Type' => 'application/json',
        'Accept'       => 'application/json'
    ],
    // 'body' => '{ "drink": "coffe" }',
    'json' => [
        'drink' => 'coffe'
    ]
]);

File: test/fixtures/r-httr2/post_json.r
library(httr2)

request("http://localhost:28139") |>
  req_method("POST") |>
  req_headers(Accept = "application/json") |>
  req_body_raw(
    '{ "drink": "coffe" }',
    type = "application/json"
  ) |>
  req_perform()
